pkg/config/testdata/TestAuthDiff/local_enabled_and_disabled.diff

```diff
@@ -3,7 +3,7 @@ diff remote[auth] local[auth]
 +++ local[auth]
 @@ -1,14 +1,14 @@
  enabled = false
--site_url = ""
+ site_url = "http://127.0.0.1:3000"
 -additional_redirect_urls = ["https://127.0.0.1:3000", "https://ref.supabase.co"]
```

Comment:
**question**
Shouldn't we still have the `https://ref.supabase.co` in the diff ?

Reply:
I updated the default additional redirect to empty array, which is why its diff is changed.